gobblin-audit/src/main/java/gobblin/audit/values/auditor/ValueAuditGenerator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.values.auditor;
+
+import gobblin.audit.values.policy.column.ColumnProjectionPolicy;
```

Comment:
None :)  
Yinan suggested me a while back that we should keep it consistent across Gobblin since we have listed it as our codestyle: https://github.com/linkedin/gobblin/wiki/CodingStyle (look for pt. 2 Import Statements)